tests/test_experiment.py

```diff
@@ -15,7 +17,8 @@ def klass(self):
         return Experiment
 
     @pytest.fixture
-    def exp(self, klass):
+    def exp(self, active_config, klass):
```

Comment:
Is this change needed?

Reply:
No, forgot to remove that after an experiment didn't pan out. Removed.